spec/models/post_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  first_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+  subject { Post.create(author: first_user, title: 'Hello', text: 'This is my first post') }
+  it 'should be inheritanc of' do
+    expect(Post < ApplicationRecord).to eq(true)
+  end
+  before { subject.save }
+  it 'title should be equle to Hello' do
+    expect(subject.title).to eq('Hello')
+  end
+
+  it 'text should be equal to This is my first post' do
+    expect(subject.text).to eq('This is my first post')
+  end
+
+  it 'author shold be equla to first_user' do
+    expect(subject.author).to eq(first_user)
+  end
+
+  context 'Check all Validation' do
+    it 'auther should be presence' do
+      author = Post.new(title: 'Hello')
+      expect(author).to_not be_valid
+      expect(author.errors[:author]).to include('must exist')
+    end
+
+    it 'text of post shoud be equ to My name ...' do
+      user2 = User.new(name: 'girma', posts_counter: 1)
+      user2.save
+      author = Post.new(author: user2, title: 'Hello', text: 'My name is girma')
+      expect(author.text).to eq('My name is girma')
+    end
+
+    it 'title should be  equla to Hello' do
```

Comment:
- [ ] As requested by the previous code reviewer please 🙏🏽 correct the `it` description to understand what the test does. May you want to say "equal"?

> Kindly correct your it blocks it hard to understand what your test cases do